ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -319,7 +319,8 @@ tests:
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential
+      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
```

Comment:
The job is for testing `none` capability on purpose, ideally we should not enable any additional caps, while because  `MachineAPI` and `CloudCredential` has to be enabled because the IPI install has to require them.

While for the added `CCM` and `Ingress`, is that the same case? The IPI install has to require them because of some known limiation?

Reply:
CCM can only be disabled on "none", "baremetal" or "external" platform with ccm none clusters, so has to be enabled on cloud clusters.
Ingress is always enabled on ocp, only disabled on hypershift